<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Reinhold Kainhofer &lt;reinhold@kainhofer.com&gt;</author>
 <comment>Configuration page for the yearly print mode.</comment>
 <class>CalPrintYearConfig_Base</class>
 <widget class="QWidget" name="CalPrintYearConfig_Base">
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Print year options:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mDateRangeGroup">
     <property name="title">
      <string>Date Range</string>
     </property>
     <property name="selectedId" stdset="0">
      <number>-1</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="mYearLabel">
          <property name="text">
           <string>&amp;Year:</string>
          </property>
          <property name="buddy">
           <cstring>mYear</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mYear">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>2500</number>
          </property>
          <property name="value">
           <number>2012</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mSecurity">
     <property name="title">
      <string>Security Exclusions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="mExcludeConfidential">
        <property name="toolTip">
         <string>Check this option to exclude items that have their Access level set to &#8220;Confidential&#8221;</string>
        </property>
        <property name="text">
         <string>Exclude c&amp;onfidential</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mExcludePrivate">
        <property name="toolTip">
         <string>Check this option to exclude items that have their Access level set to &#8220;Private&#8221;</string>
        </property>
        <property name="text">
         <string>Exclude pri&amp;vate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGeneralGroup">
     <property name="title">
      <string>General</string>
     </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="mPagesLabel">
         <property name="text">
          <string>Num&amp;ber of pages:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="buddy">
          <cstring>mPages</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="mPages"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mSubDaysLabel">
         <property name="text">
          <string>Show &amp;sub-day events as:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="buddy">
          <cstring>mSubDays</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="mSubDays">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Text</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Time Boxes</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mHolidaysLabel">
         <property name="text">
          <string>Show &amp;holidays as:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="buddy">
          <cstring>mHolidays</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="mHolidays">
         <item>
          <property name="text">
           <string>Text</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Time Boxes</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" colspan="2">
        <widget class="QCheckBox" name="mPrintFooter">
         <property name="toolTip">
          <string>Print a datetime footer on each page</string>
         </property>
         <property name="whatsThis">
          <string>Check this box if you want to print a small footer on each page that contains the date of the print.</string>
         </property>
         <property name="text">
          <string>Print &amp;Footer</string>
         </property>
        </widget>
       </item>
      </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
